.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,7 +24,7 @@ before and after the PR.-->
 - [ ] All tests passed (See
   [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
```

Comment:
```suggestion
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
```
I think making this text be all on the same line would fix the display issue I mentioned above :crossed_fingers:

Reply:
Done, thanks! a62736ee5097160bf004f21d785c8ef46e3e4429